### ISSUE TITLE
Map doesn't have bunker. Version 0.14.6

### ISSUE THREAD

**Issue by Ladysarajane** [GreatOrator/FallCraft-Map#33]
Just downloaded the pack as a new instance, downloaded and installed the Colorado map and when I started it, I started outside in wilderness with no bunker.

**Comment by GreatOrator** [comment]
What were the coordinates?

**Comment by Ladysarajane** [comment]
I started off at x:2490, z:3702, y: 91

**Comment by GreatOrator** [comment]
Can you post a picture of where you started please?  Those are close to the correct coordinates, but curious.  I just checked the map and everything is set correctly.

**Comment by Ladysarajane** [comment]
Here you go
![2018-07-04_09 06 03](https://user-images.githubusercontent.com/34258851/42278836-faa20c38-7f69-11e8-8731-a70e2a16b393.png)
![2018-07-04_09 06 12](https://user-images.githubusercontent.com/34258851/42278837-fab17024-7f69-11e8-99d5-a0b11f9da46e.png)
![2018-07-04_09 06 20](https://user-images.githubusercontent.com/34258851/42278838-fac04928-7f69-11e8-8b42-9e749a2473bd.png)

**Comment by Ladysarajane** [comment]
Just to add to this, When I download this to a new instance in twitch, I get the 0.9.1 instance. To get the correct version I have to go to the profile of Fallcraft and download the version there.

**Comment by Ladysarajane** [comment]
Okay after messing with it some more this morning these are the conclusions I have come to.
Twitch DOES not have the newest update to download, you download version 0.9.1, I had to go into profile screen and download the correct version there.  Which is odd, because my old profile has a thing to update it. I will update that profile and see if it updates to the correct version. 

As in starting, I did find the bunker. But the door is closed. I can't get in. 
![2018-07-04_09 37 53](https://user-images.githubusercontent.com/34258851/42280738-95d39cd0-7f6f-11e8-80ce-3d02d6526361.png)

It is like I did start in the correct place, but at the wrong "Y" level

**Comment by GreatOrator** [comment]
Yeah, was just going to say the location is correct save for the Y level.  Looks like minecraft performing it's standard random player placement again.  We may need to look into using perfect spawn again.

If you tp down about 15 blocks? i think it is you will be inside from that point.  or since you are at the front door, tp forward about 5 blocks.

**Comment by Ladysarajane** [comment]
I did TP in the bunker using x:2498 z:3712 y;66 and I think that is suppose to be the spawn point that was intended if I remember from before. ( I went into creative mode, broke into the bunker found the coordinates started a new world and tp to it. )
